packages/terra-overlay/src/Overlay.scss

```diff
@@ -0,0 +1,68 @@
+/* stylelint-disable selector-class-pattern  */
+// stylelint-disable declaration-property-value-blacklist
+:local {
+  .overlay-container {
+    outline: none;
+    position: relative;
+  }
+
+  .overlay {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    opacity: 0.8;
+    outline: none;
+    overflow: hidden;
+    z-index: 100;
+  }
+
+  .clear {
+    background: rgba(255, 255, 255, 0.001);
+  }
+
+  .dark {
+    background: rgba(28, 31, 33, 0.85);
+
+    .content {
```

Comment:
.content { } is setting the font color to white when a dark theme is applied. Otherwise the color is black.